sdk/eventhub/azure-eventhub/samples/async_samples/authenticate_with_named_key_credential_async.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to demonstrate utilizing AzureNamedKeyCredential to authenticate with Event Hubs.
+"""
+
+# pylint: disable=C0111
+
+import os
+import time
+from azure.core.credentials import AzureNamedKeyCredential
```

Comment:
Is this the same as importing `AzureNamedKeyTokenCredentialAsync` from `eventhub.aio` and using that to authenticate?

Reply:
AzureNamedKeyTokenCredentialAsync is not exposed to the public - it's an internal wrapper which we use to change the AzureNamedKeyCredential into a token credential